src/utils/fetchData.ts

```diff
@@ -0,0 +1,53 @@
+import { Buffer } from 'buffer';
+
+export function fetchText(uri?: string): Promise<string | null> {
+  if (uri && uri.startsWith('data:')) {
```

Comment:
Quick question: Can base64 file be fetched from remote? If yes, then it still would not work since it's base64 or fetch

Reply:
It has been updated.